README.md

```diff
@@ -23,6 +23,29 @@ _For additional improvements when fully using the ember-strict-resolver monkey p
 Ember.Registry.prototype.normalize = function (i) { return i; }
 ```
 
+## Migration
+
+Migrating away from use the _ember-resolver/classic_ can be done in piecemeal by supporting a sub-set of the old resolution formats.
+
+```js
+> _app/resolver.js_
```

Comment:
I think this makes the syntax highlighting a bit better (the `> ` bit throws it off):

```suggestion
// app/resolver.js
```